modules/sc-mesh-secure-deployment/src/nats/scripts/cli_settings_request.py

```diff
@@ -3,27 +3,80 @@
 import json
 import config
 
-
 async def main():
     # Connect to NATS!
     nc = await client.connect_nats()
-    cmd_dict = {"api_version": 1, "ssid": "test_mesh", "key": "1234567890",
-                "ap_mac": "00:11:22:33:44:55", "country": "FI", "frequency": "5220",
-                "frequency_mcc": "2412", "routing": "batman-adv", "priority": "long_range",
-                "ip": "192.168.1.2", "subnet": "255.255.255.0", "tx_power": "5", "mode": "mesh",
-                "role": f"{config.MODULE_ROLE}"}
+    cmd_dict = {
+        "api_version": 1,
+        "role": f"{config.MODULE_ROLE}",  # sleeve, drone, gcs
+        "radios": [
+            {
+                "radio_index": "0",
+                "ssid": "test_mesh2",
+                "key": "1234567890",
+                "ap_mac": "00:11:22:33:44:55",
```

Comment:
I see it was in the previous version of the file already but if we are writing this again then maybe it is the time to ask: what is ap_mac for? Can it be removed if not used?

Reply:
ap_mac is used for wpa_supplicant bssid.

Reply:
It's named as bssid in wpa supplicant.  Again this is not related to multiradio implementation.  It has been almost 3 years like and tested to be working with MP.